raimitigations/dataprocessing/imputer/imputer.py

```diff
@@ -71,20 +71,21 @@ def _set_columns_to_impute(self):
     # -----------------------------------
 
     def _reset_columns_to_impute(self, df: Union[pd.DataFrame, np.ndarray]):
```

Comment:
My final comment (I promise): please add the docstrings even for the private methods that you created. Right now they all make sense to us, but in a few months, we might struggle to remember what this function does. This applies to all new functions that you created. Also, for methods that have a return value, I noticed that you added the return type using typehint, but I think you should also add the ``:return`` and ``:rtype`` keys in the docstrings of these methods to add a brief description of what you are returning (``:return``) and the type of what you are returning (``:rtype``).

Reply:
Yes yes, good call! Thank you!